types/Modal.svelte.d.ts

```diff
@@ -86,6 +87,12 @@ export interface ModalProps {
    */
   show?: Component | null;
 
+  /**
+   * ID to reference the modal in JavaScript/CSS
+   * @default null
+   */
+  id?: string | null;
+
```

Comment:
Are those auto-generated via `npm run build`? If so, after the above changes you'd have to run the script again to pick up the adjusted text.